### DISCUSSION TITLE
Harassment controls

### DISCUSSION THREAD

**Discussion by nikclayton** [pachli/pachli-android#864] [Ideas]
This discussion thread is for public feedback on the anti-harassment controls proposed in [Anti-harassment controls in Pachli 2.9.0](https://pachli.app/pachli/2024/08/02/harassment-controls.html).

If you prefer to leave private feedback please e-mail team@pachli.app.

**Comment by nikclayton** [comment]
I'm going to use this post to track feedback I've received on the idea from elsewhere.

- Ignoring filters in search results
   - One user reports they would expect the account level filters to apply to search results too.
- On ignoring filters on accounts with roles
   - The roles associated with an account may not be public / appear in the API result. If they are present they can be used. If not, the account (if present) in the [server's contact information](https://docs.joinmastodon.org/entities/Instance/#contact-account) should be exempted from the filtering.

**Comment by aznotas** [comment]
Hello! I saw your blog post and I want to share an article about the problem with fediverse for those who under 18

https://blog.hellbeast.eu.org/Fedi%20fucking%20sucks%20%28as%20a%20minor%29.html

**Reply by nikclayton** [reply]
Thanks for the link. While I can see it's a problem for the author, it's somewhat orthogonal to the wider harassment that (predominantly BIPOC) individuals face. I'll keep it in mind, but I doubt the specific anti-harassment controls I have in mind will be useful in that use case.

I could see that providing additional "intelligence" about an account might be helpful for this though.

For example, if you're looking at the list of suggested accounts Pachli tells you when they last posted, and how frequently they posted.

Phanpy tells you what percentage of their recent posts were replies, boosts, etc.

I could see an extension of that idea that e.g., told you want percentage of an account's recent posts had content warnings; or showed you the hashtags they've used most frequently. Then you could get a quick idea of the sort of content the account typically posts.

**Comment by nikclayton** [comment]
https://github.com/pachli/pachli-android/issues/925 is the tracking issue for the work.
